src/pages/FrontPage.scss

```diff
@@ -0,0 +1,83 @@
+@import '../assets/css/fonts';
+
+.content {
+  margin-top: 75px;
+}
+
+.column {
+  display: inline-block;
+  vertical-align: top;
+}
+
+.leftColumn {
+  composes: column;
+  width: 25%;
+}
+
+.rightColumn {
```

Comment:
Jeg har ingen sterke meninger om dette, ettersom jeg ikke helt vet hvordan jeg bruker rebase. Men jeg leste én setning i artikkelen: "If you and your team are not familiar with, or don't understand the intricacies of rebase, then you probably shouldn't use it. In this context, always merge is the safest option." For min del betyr det at vi burde holde oss til merge. Men om dere andre kan rebase, så er jeg mer enn villig til å lære, dersom dere mener det er en bedre strategi.

Men jeg forstod heller ikke det som stod på toppen (punkt 3), der han sier følgende:
Review is done and ready to be integrated into the target branch. Congratulations! You‘re about to delete your feature branch. Given that other developers won’t be fetch-merging in these changes from this point on, this is your chance to sanitize history. At this point you can rewrite history and fold the original commits and those pesky ‘pr rework’ and ‘merge’ commits into a small set of focussed commits. Creating an explicit merge for these commits is optional, but has value. It records when the feature graduated to master.

Hvis jeg forstår ham rett, så er det altså ganger da det er "opplagt" at man skal rebase? Tror jeg trenger et git-kurs...